File: techila/examples/java/peach/Tutorial/2_parameters/LocalFunction.java
// Copyright 2014 Techila Technologies Ltd.

import java.util.List;
import java.util.Vector;

public class LocalFunction {

    public List<Integer> localFunction(int multip, int amount) {
        Vector<Integer> result = new Vector<Integer>();
        for (int i = 1; i < amount + 1; i++) {
            result.add(multip * i);
        }
        return result;
    }

    public void run() {

        List<Integer> result = localFunction(2, 5);

        System.out.println(result);
    }

    public static void main(String[] args) {
        LocalFunction lf = new LocalFunction();

        lf.run();
    }
}

File: techila/examples/Matlab/Tutorial/2_parameters/local_function.m
function result = local_function(multip,loops)
% This function can be executed locally on the End-Users computer. The
% function consists of a loop structure, where the value of the loop
% counter is multiplied with a fixed value.
%
% To use: result = local_function(multip,loops)
%
% multip = value of the multiplicator
% loops  = number of iterations

% Copyright 2010-2013 Techila Technologies Ltd.

for i = 1:loops
    result(i) = multip * i;
end

